WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -334,10 +341,18 @@ private extension TotalsView {
                     return .primary
                 }
             }
-        case .cash:
-            return PaymentViewLayout(backgroundColor: backgroundColor,
-                                     topPadding: nil,
-                                     bottomPadding: nil)
+        case .cash(let cashPaymentState):
+            switch cashPaymentState {
+            case .collectingCash:
+                return PaymentViewLayout(backgroundColor: backgroundColor,
+                                         topPadding: 0,
+                                         bottomPadding: nil,
+                                         sidePadding: 0)
+            case .paymentSuccess:
+                return PaymentViewLayout(backgroundColor: backgroundColor,
+                                         topPadding: nil,
+                                         bottomPadding: nil)
+            }
```

Comment:
🗒️ Note for reviewer: with the page header component, the cash view shouldn't have any top/bottom padding anymore. Because the payment success & cash payment share the same state here, the success state needs to maintain the previous `PaymentViewLayout` to have spacers at the top and bottom.